src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1105,8 +1105,15 @@ class LeVADocumentUseCase extends DocGenUseCase {
 
         def documentType = DocumentType.DTR as String
 
-        def uri = this.createOverallDocument("Overall-Cover", documentType, metadata, null, this.getWatermarkText(documentType))
-        this.updateJiraDocumentationTrackingIssue(documentType, "A new ${documentTypeName} has been generated and is available at: ${uri}.")
+        // Same behaviour to get sections not done as DTR
+        def sections = this.jiraUseCase.getDocumentChapterData(documentType)
+        if (!sections) {
+            sections = this.levaFiles.getDocumentChapterData(documentType)
```

Comment:
There is no fall-back solution for overall documents available on the file system.

Reply:
The sections are from DTR and TIR (to get status in Overall). @metmajer Do I maintain only the jira query?

Reply:
@hrcornejo the marker here is misleading. I was referring to line 1111, where you fall back to a file-based approach that does not exist for overall documents AND that does not have a status for a document chapter.

Reply:
Ok